### PR TITLE
update version and changelog

### PR DESCRIPTION
**Pull request by Blackbaud-TimPepper**

### CODE REVIEW

**Comment by Blackbaud-TimPepper** [comment]
This won't pass until [#42](https://github.com/blackbaud/skyux-builder-plugin-stache/pull/42) is pushed through, due to version dependency.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# 2.11.1 (2018-08-03)
+# 2.11.2 (2018-08-03)
```
Comment (Blackbaud-BrandonJones) [inline CHANGELOG.md]: We would need to keep this entry and add a new one above. 

```
# 2.11.2 (2018-08-07)

- Updated verion number for `skyux-builder-plugin-stache` to `1.5.1` to address a bug affecting windows users attempting to access the global Stache application data.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 
+# 2.11.1 (2018-08-07)
```
Comment (Blackbaud-BrandonJones) [inline CHANGELOG.md]: `2.11.2`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+- Addressed a Windows compatibility issue with `skyux-builder-plugin-stache` version `1.5.0`, updated to version `1.5.1`.
```
Comment (Blackbaud-BrandonJones) [inline CHANGELOG.md]: This doesn't read very well.  
```
- Updated the `skyux-builder-plugin-stache` to `1.5.1` to address a compatibility issue with Windows accessing nested global json data.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
 
+# 2.11.2 (2018-08-07)
+- Addressed a Windows compatibility issue with `skyux-builder-plugin-stache` version `1.5.0`, updated to version `1.5.1`.
```
Comment (Blackbaud-BrandonJones) [inline CHANGELOG.md]: This doesn't read very well, and we should have some mention of the issue.  Might not hurt to just link to the plugin change log if you don't want to call out the issue here.
```
- Updated the [skyux builder plugin stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md) to `1.5.1`.

**Pull request merged**